templating.yaml

```diff
@@ -0,0 +1,108 @@
+# Configure some defaults that will probably be common
+# to each package
+defaults:
+  user: prometheus
+  group: prometheus
+  package: '%{name}-%{version}.linux-amd64'
+  tarball: '{{URL}}/releases/download/v%{version}/{{package}}.tar.gz'
+  build_steps:
+    - spec
+    - unit
+    - init
+  prep: |
+    %setup -q -n {{package}}
+  build: |
+    /bin/true
+  install: |
+    mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus
+    install -D -m 755 %{name} %{buildroot}%{_bindir}/%{name}
+    install -D -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/default/%{name}
+    %if 0%{?el5}
+    install -D -m 644 %{SOURCE3} %{buildroot}%{_initrddir}/%{name}
+    %else 
+        %if 0%{?el6} 
+        install -D -m 644 %{SOURCE3} %{buildroot}%{_initddir}/%{name}
+        %else
+        install -D -m 644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
+        %endif
+    %endif
+  pre: |
```

Comment:
I don't see a `pre` in the old spec files

Reply:
This should have been an empty comment. This was incase we needed to populate this at a later date for some reason